server/src/server.ts

```diff
@@ -464,6 +464,82 @@ process.on("message", (msg: m.Message) => {
           }
         }
       }
+    } else if (msg.method === "custom:rescript-vscode.create_interface") {
```

Comment:
Is there a way not to use a custom language-server command? Just a vscode plugin-specific API? Or is this the idiomatic way of making things work?

Reply:
I guess all the work could be done on the client side which would make the message passing obsolete.
OTOH if such a process would take longer, I guess it would be better to have it run in a separate process which would be the server.

Reply:
Did the idiomatic approach now, and created a request type `createInterfaceRequest` for it.